apiserver/action/action.go

```diff
@@ -229,7 +229,7 @@ func (a *ActionAPI) ListCompleted(arg params.Entities) (params.ActionsByReceiver
 }
 
 // Cancel attempts to cancel enqueued Actions from running.
-func (a *ActionAPI) Cancel(arg params.Entities) (params.ActionResults, error) {
+func (a *ActionAPI) Cancel(arg params.Actions) (params.ActionResults, error) {
```

Comment:
Ah good, the change was prompted by the fact the the client sends `Actions`.

 and the agent was failing silently in de-serializing the `Actions` to `Entities`. I should change the client to send Entities instead of changing the agent to accept Actions. I totally agree!